src/main/scala/vexiiriscv/tester/TestBench.scala

```diff
@@ -456,7 +459,7 @@ object TestBench extends App{
     genConfig.includeSimulation
 
     val simConfig = SpinalSimConfig()
-    simConfig.withFstWave
+    simConfig.withVcdWave
```

Comment:
VCD wave are very dangerous as a default, as they can easily blow up a hard drive very first
```suggestion
    simConfig.withFstWave
```

Reply:
I guess overall, we would need to add all of those through argument / conifiguration / envrionnement variables ^^

I also have a lot the habit to modify / comment code in order to fit my specific needs, but overall, that doesn't scale up well XD

Reply:
that's why I often use this one: https://github.com/andreasWallner/spinalStuff/blob/90604fce38fab63d7a366e91a158b6e6f03e56e7/src/main/scala/andreasWallner/sim/package.scala#L268 ;-)
(though I'm not that happy with the name anymore)